Source/WebCore/loader/DocumentWriter.cpp

```diff
@@ -296,7 +296,9 @@ void DocumentWriter::end()
     // http://bugs.webkit.org/show_bug.cgi?id=10854
     // The frame's last ref may be removed and it can be deleted by checkCompleted(), 
     // so we'll add a protective refcount
+IGNORE_GCC_WARNINGS_BEGIN("stringop-overflow")
```

Comment:
It is refcounting, there is no string operation involved here?? What's this stringop-overflow about?
If stringop-overflow checks are busted in gcc, they it may be nicer to disable them globally rather than silencing random bogus warnings?

Reply:
> It is refcounting, there is no string operation involved here?? 

Yup.

> What's this stringop-overflow about?

Absolutely no clue. It seems outrageously off-base. If there's anything wrong here, I don't think we'll discover it from [this warning message](https://bugs.webkit.org/show_bug.cgi?id=239340#c0).

> If stringop-overflow checks are busted in gcc, they it may be nicer to disable them globally rather than silencing random bogus warnings?

I did consider this, and [I've done it for multiple other security-relevant warnings recently](https://github.com/WebKit/WebKit/blob/f8fdb68559b5cdf18d81101a1abbc1adc265b752/Source/cmake/WebKitCompilerFlags.cmake), including the similar -Wstringop-overread, after deciding the warnings were consistently too far off-base. But currently there are only two other places in WebKit that trip -Wstringop-overflow. I'd say the benefit of keeping it on probably outweighs the cost of three IGNORE_GCC_WARNINGS_BEGIN/END, at least for now, but if we start to see a bunch more cases like this in the future then we can always revisit.

Reply:
I just noticed two more of these... reported https://bugs.webkit.org/show_bug.cgi?id=240596.